src/middlewared/middlewared/plugins/reporting/events.py

```diff
@@ -208,7 +209,18 @@ def run_sync(self):
             stats_time = time.time()
             for i in glob.glob('/sys/class/net/*/statistics'):
                 iface_name = i.replace('/sys/class/net/', '').split('/')[0]
-                data['interfaces'][iface_name]['speed'] = last_interface_speeds['speeds'].get(iface_name)
+                if iface_name.startswith(tuple(netif.INTERNAL_INTERFACES)):
```

Comment:
Can you please add this as a method? It seems that it can be used in other middleware components?